db/migrate/20190523202200_add_token_to_users.rb

```diff
@@ -1,5 +0,0 @@
-# class AddTokenToUsers < ActiveRecord::Migration[5.2]
```

Comment:
added this field in the devise_create_user migration instead of doing it seperately